guide/01-getting-started/using-the-jupyter-lab-environment.ipynb

```diff
@@ -36,21 +36,55 @@
     "\n",
```

Comment:
<p>If "Rebuild" failed in the jupyter lab env, user is recommended to use command line in the server env to have the "build" done.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/745/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>